### ISSUE TITLE
Attendees

### ISSUE THREAD

**Issue by dilyanpalauzov** [dmfs/opentasks#713]
OpenTasks information can be mapped to VTODO, and the latter can contain ATTENDEEs.

* Support ATTENDEEs for tasks.

**Comment by dmfs** [comment]
Right. The provider has some initial support for this: https://github.com/dmfs/opentasks/blob/76dbaa7aed46bbbc955c568ff89012932dbcf89e/opentasks-contract/src/main/java/org/dmfs/tasks/contract/TaskContract.java#L1444
This is not finalized though.

**Comment by dilyanpalauzov** [comment]
It would be nice if DavX5 can store in the tables of OpenTasks for the beginning the attendees so that downloading a VTODO from a CalDAV server with attendees, editing in OpenTasks and synchronizing/updating will not silently delete the participants.

Adding the attendees to the UI can be done later.
